processor/transaction/schema.go

```diff
@@ -135,74 +135,74 @@ var transactionSchema = `{
     "description": "Data captured by an agent representing an event occurring in a monitored app",
     "properties": {
         "context": {
-              "$schema": "http://json-schema.org/draft-04/schema#",
-  "$id": "doc/spec/context.json",
-  "title": "Context",
-  "description": "Any arbitrary contextual information regarding the event, captured by the agent, optionally provided by the user",
-  "type": ["object","null"],
-  "properties": {
-    "custom": {
-      "description": "An arbitrary mapping of additional metadata to store with the event.",
-      "type": ["object", "null"],
-      "regexProperties": true,
-      "patternProperties": {
-        "^[^.*\"]*$": { }
-      },
-      "additionalProperties": false
-    },
-    "response": {
-        "type": ["object", "null"],
-        "properties": {
-            "finished": {
-                "type": ["boolean", "null"]
+                "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "doc/spec/context.json",
```

Comment:
I can `make fmt` to update this file, but it seems to mess up a few indentation

Reply:
I don't think `make fmt` cleans up json files. It's the `.editorconfig` which should take care of the indentation: https://github.com/elastic/apm-server/blob/master/.editorconfig#L10

But that does not do any linting.

Reply:
This isn't a JSON file, but a go file which was generated automatically by running `make fmt`.

Reply:
I ran it because the tests were failing and complaining about changes to the schema. I looked in the README and it suggested running `make fmt`. After I ran that the tests passed.

Reply:
The schema in this file is created automatically and does not care about intentation. If you think this should be fixed we can create an Issue to improve this.